src/action.rs

```diff
@@ -78,8 +100,33 @@ pub const fn d(layer: usize) -> Action {
     Action::DefaultLayer(layer)
 }
 
-/// A shortcut to create a `Action::KeyCode`, useful to create compact
+/// A shortcut to create `Action::MultipleKeyCodes`, useful to create compact
 /// layout.
 pub const fn m(kcs: &'static [KeyCode]) -> Action {
     Action::MultipleKeyCodes(kcs)
 }
+
+/// A shortcut to create `KeyEvent::Tap`, useful to create compact
+/// layout.
+pub const fn tap(kc: KeyCode) -> SequenceEvent {
+    SequenceEvent::Tap(kc)
```

Comment:
These shortcut are of little help compared to `use SequenceEvent::*`, and will not be so used. And with the macro we'll add, it will not really be interesting. Let's remove them for now and see at usage that helpers are common. (removing a function is a breaking change, thus we prefer to not put a function if we're not sure it's a good idea).

Reply:
This makes sense.  I'll remove them.